src/main/java/randoop/resource/MethodListGen.java

```diff
@@ -0,0 +1,182 @@
+   *
+   * @param args command line arguments
+   *     <ul>
+   *       <li>args[0] - JDK .jar location as a path. Currently only tested with JDK 8.
+   *       <li>args[1] - output directory
+   *     </ul>
+   */
+  public static void main(String[] args) {
+    Path annotatedJar = Paths.get(args[0]);
+    String outputDirectory = args[1];
+
+    Path nonDetFile = Paths.get(outputDirectory, "omitmethods-defaults.txt");
+    Path sideEffectFreeFile = Paths.get(outputDirectory, "JDK-sef-methods.txt");
+    Path unparsableSideEffectFreeFile =
+        Paths.get(outputDirectory, "JDK-sef-methods-unparsable.txt");
+
+    try {
+      List<String> nonDetMethods = getAnnotatedMethodsFromJar(annotatedJar, NONDET_ANNOTATIONS);
+
+      // Randoop expects a list of omitmethods as regex.
+      if (nonDetMethods.size() > 0) {
+        System.out.println("Nondeterministic methods count: " + nonDetMethods.size());
+        try (BufferedWriter nonDetMethodWriter = Files.newBufferedWriter(nonDetFile, UTF_8)) {
+          nonDetMethodWriter.write(OMIT_METHODS_FILE_HEADER);
+          for (String method : nonDetMethods) {
+            nonDetMethodWriter.write("^" + Pattern.quote(method) + "$");
+            nonDetMethodWriter.newLine();
+          }
+        }
+      } else {
+        System.out.println("No nondeterministic methods found. Not writing to file.");
+      }
+
+      List<String> sideEffectFreeMethods =
+          getAnnotatedMethodsFromJar(annotatedJar, SEF_ANNOTATIONS);
+
+      // There are some fully qualified signatures that Randoop cannot parse.
+      // We will separate these into two files, the default (parsable) file and
+      // the non-parsable file.
+      if (sideEffectFreeMethods.size() > 0) {
+        System.out.println("Total side effect free methods count: " + sideEffectFreeMethods.size());
+        try (BufferedWriter sideEffectMethodWriter =
+                Files.newBufferedWriter(sideEffectFreeFile, UTF_8);
+            BufferedWriter unparsableSideEffectMethodWriter =
+                Files.newBufferedWriter(unparsableSideEffectFreeFile, UTF_8)) {
+          for (String fullyQualifiedMethodSignature : sideEffectFreeMethods) {
+            try {
+              signatureToOperation(
+                  fullyQualifiedMethodSignature,
+                  VisibilityPredicate.IS_ANY,
+                  new EverythingAllowedPredicate());
+              sideEffectMethodWriter.write(fullyQualifiedMethodSignature);
+              sideEffectMethodWriter.newLine();
+            } catch (RandoopUsageError | FailedPredicateException e) {
+              System.err.println("Not parsable: " + e.getMessage());
+              unparsableSideEffectMethodWriter.write(fullyQualifiedMethodSignature);
+              unparsableSideEffectMethodWriter.newLine();
+              e.printStackTrace(new PrintWriter(unparsableSideEffectMethodWriter));
+              unparsableSideEffectMethodWriter.newLine();
```

Comment:
See the included unparseable methods file. It looks like the bulk of these either have the protected modifier or are static.